### PR TITLE
* #196: Added an argument validation to SqlFunctionAggregateGroupConcat builder

### LINKED ISSUE #196: Add validation in `SqlFunctionAggregateGroupConcat` builder for not allowing null argument
Add validation in `SqlFunctionAggregateGroupConcat` builder for not allowing null argument.

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #196

### CODE REVIEW

File: src/main/java/com/exasol/adapter/sql/SqlFunctionAggregateGroupConcat.java [hunk 1]
```diff
@@ -129,9 +129,16 @@ private Builder(final SqlNode argument) {
          * @return new {@link SqlFunctionAggregateGroupConcat} instance
          */
         public SqlFunctionAggregateGroupConcat build() {
+            validateArgument(this.argument);
             return new SqlFunctionAggregateGroupConcat(this);
         }
 
+        private void validateArgument(final SqlNode argument) {
+            if (argument == null) {
+                throw new IllegalArgumentException("SqlFunctionAggregateGroupConcat has a null argument.");
```
Comment (chiaradiamarcelo) [inline src/main/java/com/exasol/adapter/sql/SqlFunctionAggregateGroupConcat.java:138]: is it possible to also indicate in the logs the name of the `SqlFunctionAggregateGroupConcat` ?

Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/sql/SqlFunctionAggregateGroupConcat.java:138]: Do you mean a class name as a variable? I think we will see in the logs from where the exception comes anyway

Comment (chiaradiamarcelo) [reply src/main/java/com/exasol/adapter/sql/SqlFunctionAggregateGroupConcat.java:138]: no, I mean the name of the SQL function. For example: "The argument for the `MIN` function can not be NULL"

Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/sql/SqlFunctionAggregateGroupConcat.java:138]: It's a Group Concat function... :D So it's always Group Concat and nothing else

**Review by chiaradiamarcelo — APPROVED** [review]

**Pull request merged**